shared/src/test/scala/OptionHelperSpec.scala

```diff
@@ -0,0 +1,27 @@
+
+import org.scalatest.{FlatSpec, Matchers}
+import io.github.hamsters.OptionHelper._
+
+class OptionHelperSpec extends FlatSpec with Matchers {
+
+  "A none element of Option[String]" should "return empty string" in {
+    val optString: Option[String] = None
+    optString.orEmpty shouldEqual ""
+  }
+  "An none element of Option[Iterable]" should "return empty iterable" in {
+    val optList: Option[List[Any]] = None
```

Comment:
Any ? Why it's not T ? It must works with all the type

Reply:
Fixed that in the latest commit :)

Reply:
It does not seem to be fixed : 
https://github.com/sohum2002/hamsters/blob/issue-35/shared/src/test/scala/OptionHelperSpec.scala#L13